apps/desktop/desktop/src/main/java/bisq/desktop/main/content/user/reputation/accountAge/tab3/AccountAgeTab3Controller.java

```diff
@@ -100,7 +100,6 @@ void onClose() {
 
     public void onRequestAuthorization() {
         String signedMessage = String.valueOf(model.getSignedMessage());
-        // Question: do we still need the prefix?
```

Comment:
As long users use the current Bisq1 version the prefix is included in the copied message. Best to support both the current version and a future update in Bisq1 where the prefix is removed. Applying here a string replace for the prefix with empty string should do it.